cmd/lifecycle/rebaser.go

```diff
@@ -63,7 +63,7 @@ func (r *rebaseCmd) Privileges() error {
 			return cmd.FailErr(err, "initialize docker client")
 		}
 	}
-	if err := priv.RunAs(r.uid, r.gid); err != nil {
+	if err := priv.RunAs(r.uid, r.gid, true); err != nil {
```

Comment:
I wasn't sure if rebaser needs a user lookup, so left it as it is for now.

Reply:
It doesn't need a user lookup. Having a real user and preparing the env is only important when the phase executes buildpacks. `rebaser` is more like `analyzer/exporter` than like `creator`